include/picongpu/particles/InitFunctors.hpp

```diff
@@ -108,7 +109,26 @@ namespace picongpu
 
                 DensityFunctor densityFunctor(currentStep, idProvider->getDeviceGenerator());
                 PositionFunctor positionFunctor(currentStep, idProvider->getDeviceGenerator());
-                speciesPtr->initDensityProfile(densityFunctor, positionFunctor, currentStep);
+
+                log<picLog::SIMULATION_STATE>("initialize density profile for species %1%") % FrameType::getName();
+
```

Comment:
This looks like new functionality nut just updating of `initDensityProfile`

Reply:
This code has moved from `Particle.tpp` it was a cyclic include dependency which was only that way solvable.
There is no logic change and the function moved was only called from this place.